systemd-swap

```diff
@@ -362,8 +373,11 @@ case "$1" in
       FSTYPE=$(get_fs_type "${swapfc_path}")
       # must exists before stat()
       mkdir -p "$(dirname "${swapfc_path}")"
-      [[ $FSTYPE == btrfs ]] && { btrfs subvolume create "${swapfc_path}" \
-	      || mkdir -p "${swapfc_path}"; }
+      if [ "$FSTYPE" = btrfs ]; then
+        btrfs subvolume create "${swapfc_path}" &>/dev/null
+      else
+        mkdir -p "${swapfc_path}" 
```

Comment:
Isn't it possible here and on line 377 to just use `$swapfc_path`, skipping `""` and `{}`?

Reply:
Not quoting this would result in word splitting if `swapfc_path` contains spaces, thus passing multiple path arguments to `btrfs subvolume create`.